hack/golang-modules-update.sh

```diff
@@ -25,6 +25,5 @@ DIRS="./ ./hack/tools"
 for DIR in ${DIRS}; do
 	cd ${REPO_ROOT}/${DIR} && GO111MODULE=on GOPROXY=https://proxy.golang.org go mod download
 	cd ${REPO_ROOT}/${DIR} && GO111MODULE=on GOPROXY=https://proxy.golang.org go mod verify
-	cd ${REPO_ROOT}/${DIR} && GO111MODULE=on GOPROXY=https://proxy.golang.org go mod tidy
-	cd ${REPO_ROOT}/${DIR} && GO111MODULE=on GOPROXY=https://proxy.golang.org go mod vendor 
-done
\ No newline at end of file
+	cd ${REPO_ROOT}/${DIR} && GO111MODULE=on GOPROXY=https://proxy.golang.org go mod vendor
```

Comment:
we still need to have go mod tidy here. If there are some issues with that than we need to solve them..
This could be maybe related to the builder container which needs to be upgraded first.